HelloMixpanel/HelloMixpanelTests/HelloMixpanelTests.m

```diff
@@ -570,6 +570,21 @@ - (void)testEventTiming {
     XCTAssertNil(p[@"$duration"], @"Tracking the same event should require a second call to timeEvent.");
 }
 
+- (void)testNetworkingWithStress {
+    self.mixpanelWillFlush = NO;
+    stubTrack().andReturn(503);
+    for (int i=1; i<=500; i++) {
+        [self.mixpanel track:@"Track Call"];
+    }
+    [self flushAndWaitForSerialQueue];
+    XCTAssertTrue(self.mixpanel.eventsQueue.count == 500, @"none supposed to be flushed");
+    [[LSNocilla sharedInstance] clearStubs];
+    stubTrack().andReturn(200);
+    self.mixpanel.network.requestsDisabledUntilTime = 0;
```

Comment:
This should be reset with a `200`, might be useful to ensure the library automatically recovers.